merkledag/merkledag_test.go

```diff
@@ -115,12 +116,12 @@ func SubtestNodeStat(t *testing.T, n *Node) {
 	}
 
 	expected := NodeStat{
+		Hash:           k.B58String(),
 		NumLinks:       len(n.Links),
-		BlockSize:      len(enc),
-		LinksSize:      len(enc) - len(n.Data), // includes framing.
+		BlockSize:      bs,
+		LinksSize:      ls,
```

Comment:
Yes, I noticed this is the other definition, which was used. If there is a way to remove the offset from the marshaled n.Data, then it could be used.